src/Components/CreateImageWizard/steps/Packages/Packages.tsx

```diff
@@ -1100,14 +1195,15 @@ const Packages = () => {
         (module) => module.name === pkg.name,
       );
       isSelected =
-        packages.some((p) => p.name === pkg.name) && !isModuleWithSameName;
+        packages.some((p) => p.name === pkg.name && p.stream === pkg.stream) &&
```

Comment:
Information about active stream is already being stored in `packages` and `enabled_modules`, maybe we can work with that?

<img width="560" height="305" alt="image" src="https://github.com/user-attachments/assets/0aa9e1ab-6054-44b0-a415-78a887c78107" />

Were there any bugs with incorrect packages being selected? I though both the jumping package and jumping expandable were a problem with unstable sorting.

Reply:
while the stream matching wasn't essential for the jumping issue, it's a logical improvement that makes the selection logic more robust and precise. Both changes work together to create a better user experience.
Before: A package was considered selected if ANY package with the same name existed in the selection, regardless of stream
After: A package is only considered selected if both name AND stream match

This prevents edge cases where packages with the same name but different streams could show incorrect selection states.